Dockerfiles/mlpack-dependencies-debian/Dockerfile

```diff
@@ -44,6 +35,18 @@ WORKDIR /home/mlpack/
 # Setting up non-root user.
 USER mlpack
 
+# Installing boost from source 
+RUN wget --no-check-certificate \
+    "http://masterblaster.mlpack.org:5005/boost_1_49_0.tar.gz" \
+    && tar xvzf boost_1_49_0.tar.gz
+WORKDIR boost_1_49_0
+RUN ./bootstrap.sh --prefix=/usr \
+    --with-libraries=math,program_options,serialization,test && ./b2
+ENV BOOST_ROOT=/home/mlpack/boost_1_49_0
```

Comment:
Wouldn't it be better to install the library to /usr instead of leaving it in /home/mlpack/boost_1_49_0?  Also, I guess downloading and unpacking inside of the Dockerfile fixes the permissions issues.